AutocompleteClient/FW/Logic/Request/Builder/RequestBuilder.swift

```diff
@@ -105,6 +107,26 @@ class RequestBuilder {
         return request
     }
 
+    final func getQuizRequest(finalize: Bool) -> URLRequest {
```

Comment:
Not sure if this is the best way to handle this, but this will make it so the consumer of the client doesn't have to pass in `finalize` as an argument when calling `getQuizResults`.